straxen/plugins/event_patternfit.py

```diff
@@ -523,6 +532,8 @@ def s1_area_fraction_top_probability(aft_prob, area_tot, area_fraction_top, mode
     if do_test:
         if mode == 'discrete':
             binomial_test = binom_pmf(area_top, area_tot, aft_prob)
+            # TODO:
+            #binomial_test = binomtest(k=round(area_top), n=round(area_tot), p=aft_prob, alternative='two-sided').pvalue
```

Comment:
?

Reply:
I wanted to implement the discrete binomial test implemented in scipy instead of the one there is right now, but numba does like it

Reply:
gotcha, lets have that wait the next round then